hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -197,7 +241,7 @@ def publish_files(self, task_settings, item, publish_path):
         alembic_args.append("-file %s" % publish_path.replace("\\", "/"))
 
         # Set the root node to be exported
-        alembic_args.append("-root %s" % item.get_property("lod_full_name"))
+        alembic_args.append("-root %s" % item.properties.fields["node"])
```

Comment:
Please use lod_full_name. This was edited on purpose to avoid errors when multiple nodes have the same name.
Please ensure you always take the changes from master (git pull, git rebase master) before you commit your current change, to avoid accidentally committing stale data.